pkg/controller/appdefinition/generation.go

```diff
@@ -7,6 +7,7 @@ import (
 
 func UpdateGeneration(req router.Request, resp router.Response) error {
```

Comment:
Well, I guess we should rename this?
```
UpdateObservedFields
```
?

Reply:
Yep, good call.